### ISSUE TITLE
"voicefixer" is not recognized as an internal or external command

### ISSUE THREAD

**Issue by Scudoxx** [haoheliu/voicefixer#30]
Hello!
I plan to use voicefixer from the command line. In accordance with the instructions, I produce the following commands on the command line:

1) pip install voicefixer==0.1.1
2) git clone https://github.com/haoheliu/voicefixer.git
3) cd voicefixer

all these commands run without any error, ok.
But then, as soon as I try to run the «voicefixer» command (for example, this command):
voicefixer --infile /path/to/input.wav --outfile /path/to/output.wav
a message is displayed:
“voicefixer” is not recognized as an internal or external command…and so on
even when I just write one word “voicefixer” on the command line, the same message is displayed
As I understand it, this executable file cannot be found for some reason.
How to fix it? I use Windows 10, I also installed the recommended WGET.

**Comment by musicalman** [comment]
You are correct, there is no executable to run. But you can still use voicefixer without one.
By the way, I don't think cloning the repository is required for command line testing. I cloned it like you, but I later deleted the files and it still runs fine.
To run it, I did the following:
Go to your Python scripts folder, it should be something like AppData\Local\Programs\Python\Python39\Scripts
You should see a file called voicefixer with no extension. This is the python script for the executable.
I renamed this file to vf.py. Any name will do, just don't use the name voicefixer.py as you'll get circular references and it won't run.
Now in the command prompt, just do something like:
vf.py --infile input.wav --outfile output.wav
And processing should commence.

**Comment by Scudoxx** [comment]
Thank you for your reply, musicalman! I finally managed to run the VF script. But the following error occurred (picture):

![image](https://user-images.githubusercontent.com/112695105/188259126-a4e4302f-c968-45c3-8942-26046b93d089.png)

By the way, I am not strong in programming, and I use only those instructions that have been described on the main page. I'm a musician and I was going to use the VM to restore the voices. It is a pity that this program does not have a convenient graphical user interface.

**Comment by haoheliu** [comment]
@Scudoxx Sorry for the inconvenience. Now the bug on the windows machine should be fixed. Hope you enjoy it.
And Thanks @musicalman for the comments!

**Comment by Scudoxx** [comment]
Sorry to bother you, but apparently I'm not too lucky as nothing worked for me again:
![Без имени-1](https://user-images.githubusercontent.com/112695105/188277659-8f5a82da-8ebc-445f-82e8-95c6d670510d.jpg)

As for the second method (streamlit), which for me was the most simple and preferred, also refuses to work:

![234](https://user-images.githubusercontent.com/112695105/188277662-fdc9be3d-bf14-4490-93f2-c5c7d32e4b38.jpg)

If any of these methods ever work, I'll be happy :D
